source/plugin/plugin-class.rst

```diff
@@ -62,21 +62,32 @@ interacting with the game, such as registering commands or events.
 
 Your plugin can listen for particular events, called **state events**, to be notified about changes in the state of the
 game. In the example below, ``onServerStart()`` is called when the :javadoc:`GameStartedServerEvent` occurs; take note
-of the :javadoc:`Listener` annotation before the method.
+of the :javadoc:`Listener` annotation before the method.  
+
+The example below will log a message upon starting the server.  If your plugin is correctly loaded, you should see this message as part of the server's initialization output. 
```

Comment:
An extra space before If

Reply:
Looking at this again, it's a really long line that should be divided into two, ref: https://github.com/SpongePowered/SpongeDocs/pull/725#discussion_r205389138